controllers/rcj_soccer_referee_supervisor/referee/supervisor.py

```diff
@@ -111,60 +118,99 @@ def _update_positions(self):
             yellow (int): score of the yellow team
         """
 
-        self.setLabel(0, str(blue),
-                      # X and Y positions
-                      0.92, 0.01,
-                      # Size and color
-                      0.1, 0x0000ff,
-                      # Transparency and Font
-                      0.0, "Tahoma")
-
-        self.setLabel(1, str(yellow),
-                      # X and Y positions
-                      0.05, 0.01,
-                      # Size and color
-                      0.1, 0xffff00,
-                      # Transparency and Font
-                      0.0, "Tahoma")
-
-    def draw_time(self, time):
-        """
-        Visualize (draw) the current match time
+        self.setLabel(
+            LabelIds.BLUE_SCORE,
+            str(blue),
+            0.92,  # X position
+            0.01,  # Y position
+            0.1,  # Size
+            0x0000ff,  # Color
+            0.0,  # Transparency
+            "Tahoma",  # Font
+        )
+
+        self.setLabel(
+            LabelIds.YELLOW_SCORE,
+            str(yellow),
+            0.05,  # X position
+            0.01,  # Y position
+            0.1,  # Size
+            0xffff00,  # Color
+            0.0,  # Transparency
+            "Tahoma"  # Font
+        )
+
+    def draw_time(self, time: int):
+        """Visualize (draw) the current match time
 
         Args:
             time (int): the current match time
         """
 
-        time_str = "%02d:%02d" % (time // 60, time % 60)
-        self.setLabel(2, time_str, 0.45, 0.01, 0.1, 0x000000, 0.0, "Arial")
+        self.setLabel(
+            LabelIds.TIME,
+            time_to_string(time),
+            0.45,
+            0.01,
+            0.1,
+            0x000000,
+            0.0,
+            "Arial",
+        )
+
+    def draw_event_messages(self):
+        """Visualize (draw) the event messages from queue"""
+        rev = reversed(self.event_messages_to_draw)
+        for i, (time, msg) in enumerate(rev, start=1):
+            message = "{} - {}".format(time_to_string(time), msg)
+            self.setLabel(
+                getattr(LabelIds, "EVENT_MESSAGE_{}".format(i)),
+                message,
+                0.01,
+                0.95 - ((i-1)*0.02),
+                0.05,
+                0xffffff,
+                0.0,
+                "Arial"
+            )
+
+    def add_event_message_to_queue(self, message: str):
+        if len(self.event_messages_to_draw) >= MAX_EVENT_MESSAGES_IN_QUEUE:
+            self.event_messages_to_draw.pop(0)
+
+        self.event_messages_to_draw.append((self.time, message))
 
     def _pack_packet(
         self,
         robot_rotation: dict,
         robot_translation: dict,
         ball_translation: list,
     ):
-        """
-        Take the positions and rotations of the robots and the ball and pack
+        """ Take the positions and rotations of the robots and the ball and pack
```

Comment:
@Adman I realize this is Google's own suggestion for their docstrings but I cannot help it, it looks way worse than at its own line :)